src/org/ses/android/soap/tasks/ParticipantLoadFromCodigoTask.java

```diff
@@ -0,0 +1,65 @@
+package org.ses.android.soap.tasks;
+
+import android.os.AsyncTask;
+
+import org.ksoap2.SoapEnvelope;
+import org.ksoap2.serialization.SoapObject;
+import org.ksoap2.serialization.SoapSerializationEnvelope;
+import org.ksoap2.transport.HttpTransportSE;
+import org.ses.android.soap.database.Participant;
+
+public class ParticipantLoadFromCodigoTask extends AsyncTask<String, String, Participant> {
+
+    @Override
+    protected Participant doInBackground(String... params) {
```

Comment:
Can you explain in a comment here which parameters are needed? (e.g. param 1 = URL server)